src/llvm-final-gc-lowering.cpp

```diff
@@ -119,7 +119,7 @@ void FinalLowerGC::lowerPushGCFrame(CallInst *target, Function &F)
     builder.SetInsertPoint(&*(++BasicBlock::iterator(target)));
     Instruction *inst =
         builder.CreateStore(
-            ConstantInt::get(T_size, nRoots << 1),
+            ConstantInt::get(T_size, nRoots << 2),
```

Comment:
Should this be `JL_GC_ENCODE_PUSHARGS` as well or is there a subtle difference here?